comp/livestream/configure-scenes.yaml

```diff
@@ -0,0 +1,16 @@
+summary: Organise livestream scenes
+
+priority: major
+
+component: Competition
+
+milestone: $SRYYYY Competition Prep
+
+description: >-
+  The livestream requires a number of different scenes and views, made up of
+  static images, video feeds, overlays etc.
```

Comment:
Is this just planning what the scenes will be, or also preparing the assets to use for those which are static? (Or do we already have something for the latter?)

Reply:
We already have the majority of the assets, but we should add a task to make sure we either check them, update them, or make new ones.

Reply:
Maybe worth clarifying that? Also a link to the existing assets would be great.